src/types/module.rs

```diff
@@ -130,6 +130,7 @@ impl PyModule {
                 return Err(PyErr::api_call_failed(py));
```

Comment:
I think we also need to DECREF here? Otherwise when `PyImport_ExecCodeModuleEx` goes wrong, `cptr` is still leaked.

Reply:
Correct.

Reply:
Yes, I missed that. It's fixed now.